core/storage/suggestion/gae_models.py

```diff
@@ -1292,3 +1292,678 @@ def export_data(
                     date.isoformat() for date in model.contribution_dates]
             }
         return user_data
+
+
+class TranslationReviewStatsModel(base_models.BaseModel):
+    """Records the contributor dashboard translation review stats. There
+    is one instance of this model per (language_code, reviewer_user_id,
+    topic_id) tuple. See related design doc for more details:
+    """
+
+    # We use the model id as a key in the Takeout dict.
+    ID_IS_USED_AS_TAKEOUT_KEY = True
+
+    # The ISO 639-1 language code for which the translation reviews were
+    # made.
+    language_code = datastore_services.StringProperty(
+        required=True, indexed=True)
+    # The user ID of the translation reviewer.
+    contributor_user_id = datastore_services.StringProperty(
+        required=True, indexed=True)
+    # The topic ID of the translation reviews.
+    topic_id = datastore_services.StringProperty(required=True, indexed=True)
+    # The number of reviewed translations.
+    reviewed_translations_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The total word count of reviewed translations. Excludes HTML tags and
+    # attributes.
+    reviewed_translation_word_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The number of accepted translations.
+    accepted_translations_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The number of accepted translations with reviewer edits.
+    accepted_translations_with_reviewer_edits_count = (
+        datastore_services.IntegerProperty(required=True, indexed=True))
+    # The total word count of accepted translations. Excludes HTML tags and
+    # attributes.
+    accepted_translation_word_count = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    # The total word count of accepted translations with reviewer edits.
+    # Excludes HTML tags and attributes.
+    accepted_translations_with_reviewer_edits_word_count = (
+        datastore_services.IntegerProperty(required=True, indexed=True))
+    # The unique last_updated dates of the translation reviews.
```

Comment:
Explain the format in which the month is stored.

Reply:
Decided to drop contribution months/review months since we never use them anywhere and first contribution date and the last contribution date are sufficient to show the contribution range in the front end. If we need the exact dates, we can use the created date in suggestion model.